services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java

```diff
@@ -152,6 +153,35 @@ public Response createNews(@Context HttpServletRequest request, @ApiParam(value
     }
   }
 
+  @Path("scheduleNews")
+  @POST
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces(MediaType.APPLICATION_JSON)
+  @RolesAllowed("users")
+  @ApiOperation(value = "Schedule a news", httpMethod = "POST", response = Response.class, notes = "This schedules the news if the authenticated user is a member of the space or a spaces super manager. The news is created in staged status, after reaching a date of publication startPublishedDate, the publicationState property is set to 'published'.")
+  @ApiResponses(value = { @ApiResponse(code = 200, message = "News scheduled"),
+          @ApiResponse(code = 400, message = "Invalid query input"),
+          @ApiResponse(code = 401, message = "User not authorized to schedule the news"),
+          @ApiResponse(code = 500, message = "Internal server error") })
+  public Response scheduleNews(@Context HttpServletRequest request, @ApiParam(value = "News", required = true) News news) {
+    if (news == null || StringUtils.isEmpty(news.getSpaceId())) {
+      return Response.status(Response.Status.BAD_REQUEST).build();
+    }
+
+    String authenticatedUser = request.getRemoteUser();
```

Comment:
Refer to this [commit](https://github.com/exoplatform/news/commit/1718dea8303a911e4692830220281cab0a411d3a#diff-beada7e02f84071171fedbad6649034bff79a3f066ffebfa1283d1271dee23efR673)